### ISSUE TITLE
Is it possible to check errors when saving file only ?

### ISSUE THREAD

**Issue by KikitheKing** [nokia/RED#381]
Hello,

I'm asking if it's possible to check the code only when saving the .robot file because i have many robot scripts and some of them contains many code lines so when i'm taping some new code, Red is starting to check the code and slowdown everything (taking 90% of my CPU Core i7-6700).

Current workaround is to turn off the Red validation to get performance back but i don't like the idea.

Checking only when saving should help a lot but i cannot find if it's currently possible or not (Red 0.9.4).

BR,

**Comment by michalanglart** [comment]
Hi,

No, it cannot be currently disabled.

However RED is only validating the content of the editor when working with the file, without analysing problems in other files. Is it a big file?

**Comment by KikitheKing** [comment]
I have some big file with 1000+ lines of code.
I can see Red checking the file when taping some content and many time, it checks the file in loop multiple time even if i stop (can see potential errors blinking because i suppose every errors are reseted when starting to check).

By the way, i have just updated to 0.9.5 and i seems better (placebo effect ?).

I will let you know and will try to do a small video or a gif file.

**Comment by michalanglart** [comment]
Yeah, 0.9.5 shouldn't do any better imho. Apart from changing eclipse platform and dropping support for RF 2.9 there were some rather minor fixes.
